solutions/usaco-807.mdx

```diff
@@ -43,5 +43,61 @@ print(ans, file = open('teleport.out', 'w'))
 ```
 
 </PySection>
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+
+public class Teleportation {
+	// Function to sort values in increasing order
+	static int[] sort(int a, int b) {
+		if (a > b) {
+			return new int[] { a, b };
+		} else {
+			return new int[] { b, a };
+		}
+	}
```

Comment:
this method is really questionable
i'd prefer you do something like
```java
if (a > b) {
	int temp = a;
	a = b;
	b = temp;
}
```
and something analogous for c & d